ecosystem/typescript/sdk/src/transaction_builder/aptos_types/transaction.ts

```diff
@@ -391,6 +394,9 @@ export class TransactionPayloadScript extends TransactionPayload {
   }
 }
 
+/**
+ * @deprecated
+ */
```

Comment:
delete? we should probably delete this in upstream Rust if we can... or at least rename it to deprecated.